### PR TITLE
fix: changed constants to strings where ever needed

### LINKED ISSUE #824: Fix: Change Constants to strings
**Summary:**
We have many Constants in our app which are linked to views which might cause issues when we migrate. 
We need to convert them into strings.xml

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes: #824 
Converted the Constants to strings wherever possible

- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java [hunk 1]
```diff
@@ -206,7 +206,7 @@ public void setAccountBalance(Account account) {
         hideSwipeProgress();
 
         TransitionManager.beginDelayedTransition(homeScreenContainer);
-        mTvAccountBalance.setText(Constants.TAP_TO_REVEAL);
+        mTvAccountBalance.setText(R.string.tap_to_reveal);
```
Comment (jawidMuhammadi) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: Are you sure it is working correctly? What about if we use`getString(R.string.tap_to_reval)` here and in line numbers `120`, `132` and `165`

**Review by jawidMuhammadi — CHANGES_REQUESTED** [review]

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java (hunk 1 above)
Comment (Bhavnaharitsa) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: I checked in output. It went fine. Let me attach the screen shot

Comment (Bhavnaharitsa) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: > Are you sure it is working correctly? What about if we usegetString(R.string.tap_to_reval) here and in line numbers 120, 132 and 165

Anyway, the build has failed so i'll have to look into that too

Comment (jawidMuhammadi) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: > I checked in output. It went fine. Let me attach the screen shot

Greate, if it works :)

Comment (jawidMuhammadi) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: > > Are you sure it is working correctly? What about if we usegetString(R.string.tap_to_reval) here and in line numbers 120, 132 and 165
> 
> Anyway, the build has failed so i'll have to look into that too

You have 6 PMD violations

Comment (Bhavnaharitsa) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: How do I solve it?

Comment (jawidMuhammadi) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: Inside Android Studio in terminal run `./gradlew check` command. When it finished, go to `file:///home/travis/build/openMF/mobile-wallet/mifospay/build/reports/pmd/pmd.html` There you will find what violations you have done.

Comment (jawidMuhammadi) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: Also, there will be a link to the above file when all checks are done in the terminal. You can find it by clicking on that link. If you need any more help, feel free to message me.

Comment (Bhavnaharitsa) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeFragment.java]: Sure, thank you!

**Comment by rachittt** [comment]
Hi @Bhavnaharitsa, can you resolve the conflicts?

**Comment by shiv07tiwari** [comment]
@Bhavnaharitsa Can you please look into the build fail and conflicts.

**Pull request closed without merging**